packages/serializer/src/util.js

```diff
@@ -6,3 +6,8 @@ export const getComponentName = (str = '') => {
   const match = str.match(/^\<?([\w\.\_]+)/)
   return match && match[1]
 }
+
+export const toJS = map => {
+  if (typeof map.toJS !== 'function') return map
+  return map.toJS()
```

Comment:
Would this negate the need for the `getProps` thing I'm doing in the node rendering?

Reply:
It won't unfortunately, this is coming back from Slate schema before `getProps` is run in the editor.